### PR TITLE
[Merged by Bors] - remove node id from log and shorten session and request

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

**Review by poszu — APPROVED** [review]

**Review by fasmat — APPROVED** [review]

File: log/context.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func WithRequestID(ctx context.Context, requestID string, fields ...LoggableFiel
 // It can be used when there isn't a single, clear, unique id associated with a request (e.g.,
 // a block or tx hash).
 func WithNewRequestID(ctx context.Context, fields ...LoggableField) context.Context {
-	return WithRequestID(ctx, uuid.New().String(), fields...)
+	return WithRequestID(ctx, limit(uuid.New().String(), 10), fields...)
```
Comment (fasmat) [inline log/context.go]: NIT: this does the same thing without an extra utility function:

```suggestion
	return WithRequestID(ctx, fmt.Sprintf("%.10s", uuid.New()), fields...)
```

File: log/context.go [hunk 2]
```diff
@@ -100,8 +100,15 @@ func WithSessionID(ctx context.Context, sessionID string, fields ...LoggableFiel
...
+		return str[:n]
+	}
+	return str
+}
+
 // WithNewSessionID does the same thing as WithSessionID but generates a new, random sessionId.
 // It can be used when there isn't a single, clear, unique id associated with a session.
 func WithNewSessionID(ctx context.Context, fields ...LoggableField) context.Context {
-	return WithSessionID(ctx, uuid.New().String(), fields...)
+	return WithSessionID(ctx, limit(uuid.New().String(), 10), fields...)
```
Comment (fasmat) [inline log/context.go]: NIT: same as above

```suggestion
	return WithSessionID(ctx, fmt.Sprintf("%.10s", uuid.New()), fields...)
```

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors merge

**Comment by dshulyak** [comment]
bors merge

**Pull request closed without merging**